src/Core/Manufacturer/ManufacturerInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
@mickaelandrieu do we really need an interface here if that's a case only for one object? isn't it overkill?

Reply:
it's never overkilled when you need to uncouple classes, it's not about the quantity of code, it's about describing the direction of dependencies.

Implementation = hard bounding
Contract = Dependency Inversion